src/Illuminate/Foundation/Console/MigrateAutoCommand.php

```diff
@@ -0,0 +1,63 @@
+<?php
+
+namespace Illuminate\Foundation\Console;
+
+use Doctrine\DBAL\Schema\Comparator;
+use Illuminate\Console\Command;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Filesystem\Filesystem;
+use Illuminate\Support\Facades\Artisan;
+use Illuminate\Support\Facades\Schema;
+
+class MigrateAutoCommand extends Command
+{
+    protected $signature = 'migrate:auto {--fresh} {--seed} {--force}';
+
+    public function handle()
+    {
+        Artisan::call('migrate' . ($this->option('fresh') ? ':fresh' : null) . ($this->option('force') ? ' --force' : null));
+
+        $filesystem = new Filesystem;
+        $dir = base_path('app/Models');
```

Comment:
What if the models are no there.

Reply:
This would only work for the default model path that comes with L8. However, we could add a config value to the `database` config file, something like `model_path`?